docs/package.json

```
@@ -31,17 +31,17 @@
     "@fortawesome/fontawesome-svg-core": "^6.1.1",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/react-fontawesome": "^0.1.18",
-    "@mui/base": "5.0.0-alpha.80",
-    "@mui/docs": "^5.7.0",
-    "@mui/icons-material": "^5.6.2",
-    "@mui/joy": "5.0.0-alpha.27",
-    "@mui/lab": "5.0.0-alpha.81",
-    "@mui/material": "^5.7.0",
-    "@mui/material-next": "6.0.0-alpha.35",
-    "@mui/styled-engine": "^5.7.0",
-    "@mui/styled-engine-sc": "^5.7.0",
-    "@mui/styles": "^5.7.0",
-    "@mui/system": "^5.7.0",
+    "@mui/base": "5.0.0-alpha.81",
+    "@mui/docs": "^5.8.0",
+    "@mui/icons-material": "^5.6.3",
```

```suggestion
    "@mui/icons-material": "^5.8.0",
```
We should keep the versions consistent, no? cc @siriwatknp